src/etc/gdb_load_rust_pretty_printers.py

```diff
@@ -1,3 +1,17 @@
+# Add this folder to the python sys path; GDB Python-interpreter will now find modules in this path
+import sys
+from os import path
+self_dir = path.dirname(path.realpath(__file__))
+sys.path.append(self_dir)
+
 import gdb
 import gdb_lookup
-gdb_lookup.register_printers(gdb.current_objfile())
+
+# current_objfile can be none; even with `gdb foo-app`; sourcing this file after gdb init now works
+try:
+    current_objfile = gdb.lookup_objfile(gdb.selected_inferior().progspace.filename)
+    if current_objfile is not None:
+        gdb_lookup.register_printers(current_objfile)
+except:
```

Comment:
I think this is preferred in general to avoid catching `SystemExit` and `KeyboardInterrupt`
```suggestion
except Exception:
```

Reply:
Got it! Fixed in latest commit